service/compute/commands.go

```diff
@@ -9,6 +9,63 @@ import (
 	"github.com/databricks/databricks-sdk-go/useragent"
 )
 
+type CommandExecutorV2 struct {
+	clustersAPI  *ClustersAPI
+	executionAPI *CommandExecutionAPI
+	language     Language
+	clusterID    string
+	contextID    string
+}
+
+// Start the command execution context on a cluster and ensure it transitions to a running state
+func (a *CommandExecutionAPI) Start(ctx context.Context, clusterID string, language Language) (*CommandExecutorV2, error) {
+	executionImpl, ok := a.impl.(*commandExecutionImpl)
+	if !ok {
+		return nil, fmt.Errorf("unknown command executionimplementation: %v", a.impl)
+	}
+	// re-initializing clusters API here, so that we have less IF's in the generator
```

Comment:
I don't understand this comment. Why does the generator have anything to do with this?

Reply:
workspace client generation